project-api/ballerina-projects/src/main/java/io/ballerina/projects/utils/ProjectUtils.java

```diff
@@ -108,5 +108,14 @@ static Path createTargetDirectoryStructure(Path sourceRoot) throws IOException {
 
         return targetDir;
     }
+
+    public static String getBaloName(String org, String pkgName, String version, String platform) {
+        // <orgname>-<packagename>-<platform>-<version>.balo
+        if (platform == null || "".equals(platform)) {
+            platform = "any";
+        }
+        return org + "-" + pkgName + "-" + platform + "-" + version + ".balo";
```

Comment:
There is already a constant defined for `".balo"` in ProjectDirConstants. Lets copy that to ProjectConstants and use it.

Reply:
Used constant